sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/Introspector.scala

```diff
@@ -41,12 +46,16 @@ object Introspector {
         .toSeq
         .flatten
 
-    val dynamicRestMethods =
-      restService.methods.map(method => DynamicMethodInfo.build(method, nameGenerator, declaredEntityKeys))
+    val methodsInfo =
+      restService.methods.map(restMethod => DynamicMethodInfo.build(restMethod, nameGenerator, declaredEntityKeys))
 
-    val messageDescriptors = dynamicRestMethods.map { method =>
-      grpcService.addMethod(method.method)
-      method.descriptor
+    val messageDescriptors = methodsInfo.flatMap { methodInfo =>
+      grpcService.addMethod(methodInfo.grpcMethod)
+      // Input message descriptor can be a None. That happens when the desired type is Any.
+      // In such case, we should not add an `Any` as a message to the generated proto file
+      // And we can't properly filter it out because that's a DescriptorProto and it has no package information.
+      // Therefore, our best option is to use a None to exclude it.
+      methodInfo.inputMessageDescriptor
```

Comment:
I wish I could have found a better solution. Not obvious why it's a None. Info is spread through different parts of the code.